google/cloud/storage/CMakeLists.txt

```diff
@@ -45,7 +45,7 @@ if (TARGET storage-docs AND TARGET cloud-docs)
 endif ()
 
 include(IncludeNlohmannJson)
```

Comment:
@coryan Can we use the cmake files provided by the nlohmann json package, instead of fetching it manually via cmake?

```
$out/include
$out/include/nlohmann
$out/include/nlohmann/json.hpp
$out/lib
$out/lib/cmake
$out/lib/cmake/nlohmann_json
$out/lib/cmake/nlohmann_json/nlohmann_jsonConfig.cmake
$out/lib/cmake/nlohmann_json/nlohmann_jsonConfigVersion.cmake
$out/lib/cmake/nlohmann_json/nlohmann_jsonTargets.cmake

```

Reply:
See #2874.  We have not had time to do that, but yes, it makes sense.

Reply:
Thanks, subscribed to that issue.